app/imports/ui/layouts/App.jsx

```diff
@@ -47,6 +48,9 @@ class App extends React.Component {
             <Route path="/signout" component={Signout}/>
             <Route path="/test" component={TestContract}/>
             <ProtectedRoute path="/list" component={ListStuff}/>
+            <ProtectedRoute path="/add" component={AddStuff}/>
```

Comment:
you'll need to remove /add and /edit/:_id as @devineng repurposed them for SmartContract. Can't use the same path twice without breaking functionality.